sockeye/beam_search.py

```diff
@@ -46,7 +46,17 @@ def encode_and_initialize(self,
     def decode_step(self,
                     step_input: pt.Tensor,
                     states: List,
-                    vocab_slice_ids: Optional[pt.Tensor] = None):
+                    vocab_slice_ids: Optional[pt.Tensor] = None,
+                    target_prefix_factor: Optional[pt.Tensor] = None,
```

Comment:
Needs to be renamed to `target_prefix_factor_mask`

Reply:
Done. Thanks!